client/reader/discover/style.scss

```
@@ -12,3 +12,30 @@
 		}
 	}
 }
+
+.add-sites-tab {
```

Nit: we can use an existing selector and avoid adding new class to components.

```suggestion
.discover-navigation-v2 {
```

---

I thought it could be good to have a separate classname to call out this style, but either way works for me.

`.discover-navigation-v2` doesn't work for this because it doesn't target the elements we want, so I've pushed up a change using `.is-discover-stream`.